src/components/MovieDescription/MovieDescription.js

```diff
@@ -1,20 +1,35 @@
 import './MovieDescription.css';
+import Genre from '../Genre/Genre';
 
 const MovieDescription = ({info}) => {
+    const formatDollarString = num => {
+        if (num >= 100000000) {
+            return `$${(num/100000000).toFixed(1)} billion`
+        }
+        if (num >= 1000000) {
+            return `$${(num/1000000).toFixed(1)} million`
+        }
+        return `$${num}`
+    }
     return (
-        <div>
-            <div>
-                <h3>{info.tagline}</h3>
-                <p>{info.overview}</p>
+        <section className='MovieDescription'>
+            <div className='Movie-info'>
+                <h3 className='Tagline'>{info.tagline ? info.tagline : 'The world\'s best tagline'}</h3>
+                {info.overview && <p className='Overview'>{info.overview}</p>}
+                <div className='Genre-container'>
+                    {info.genres.map(genre => <Genre key={genre} genre={genre}/>)}
+                </div>
             </div>
-            <div>
-                <h3>Stats</h3>
-                <p>Budget: {info.budget}</p>
-                <p>Revenue: {info.revenue}</p>
-                <p>Runtime: {info.runtime} minutes</p>
-            </div>
-        </div>
+            <aside className='Stats'>
+                <h3>Stats:</h3>
+                <p>Budget: {info.budget ? formatDollarString(info.budget) : '--'}</p>
+                <p>Revenue: {info.revenue ? formatDollarString(info.revenue) : '--'}</p>
+                <p>Runtime: {info.runtime ? `${info.runtime} min` : '--'}</p>
+            </aside>
+        </section>
```

Comment:
WOW conditional rendering everything! This is awesome and looks like we will be ready to implement PropTypes very easily!